Fingers/GrooveTemplates.cpp

```diff
@@ -89,9 +89,11 @@ std::string GrooveTemplateHandler::GetGrooveDir()
 {
     std::string dir = getReaperProperty("groove_dir");
     if (dir.empty())
-    {        // Installer puts the grooves into the resource path under "Grooves"
+    {   // Installer puts the grooves into the resource path under "Data/Grooves"
```

Comment:
Will the old Grooves location still work after the latest build has been installed?

Reply:
If you changed the grooves directory to a custom location, it will keep reading from that custom directory.

If you did not change it, it switches to the new directory. The Windows installer moves the old directory contents to the new location, whereas the Linux tarball and ReaPack simply installs the new directory fresh. (And the old DMG for macOS continues doing nothing.)

Reply:
I didn't change it from _resourse directory_/Grooves and i don't install the extensions binary directly into REAPER's directory, i first extract it and then move the .dll file there manually.  Will the grooves in such a case be picked up by the extension or will i have to move them manually into the new location?

Reply:
> i don't install the extension binary directly into REAPER's directory, i first extract it and then move the .dll file there manually

Do you mean you don't run the SWS installer .exe, but instead extract the files out of it and copy them manually? (Just curious, not sure I understood that sentence right)

In that case, yes you would have to either move it into the Data subfolder where it now looks for by default, or right click > Select groove folder.

Reply:
Got it, thank you.

I do run the installer but bypassing the REAPER directory.